android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="com.example.plugin_tozny">
-    <uses-sdk android:minSdkVersion="23"
+    <uses-sdk android:minSdkVersion="26"
```

Comment:
Do you know what causes this bump. That is not great?

Reply:
Yeah so its the snippet that allows easy serializing:
```    private static DateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZZZZZ");


    @RequiresApi(api = Build.VERSION_CODES.O)
    static String recordMetaToJson(RecordMeta meta) {
        Map<String, Object> metaMap = new HashMap<>();
        metaMap.put("record_id", meta.recordId().toString());
        metaMap.put("user_id", meta.userId().toString());
        metaMap.put("writer_id", meta.writerId().toString());
        metaMap.put("created", dateFormatter.format(meta.created()));
        metaMap.put("last_modified", dateFormatter.format(meta.lastModified()));
```

Reply:
Build.VERSION_CODES.O -> Version Oreo which is android 8 and up. With android currently sitting on version 11 I assumed its relatively safe for users, but we don't have a clear directive on what version we're supporting so good callout